Menus/ObjectPickerGUI.cpp

```diff
@@ -147,7 +147,22 @@ namespace RTE {
 	void ObjectPickerGUI::SetNativeTechModule(int whichModule) {
 		if (whichModule >= 0 && whichModule < g_PresetMan.GetTotalModuleCount()) {
 			m_NativeTechModuleID = whichModule;
-			if (m_NativeTechModuleID > 0) { SetObjectsListModuleGroupExpanded(m_NativeTechModuleID); }
+			if (m_NativeTechModuleID > 0) {
+				SetObjectsListModuleGroupExpanded(m_NativeTechModuleID);
+
+				if (!g_SettingsMan.FactionBuyMenuThemesDisabled()) {
+					if (const DataModule *techModule = g_PresetMan.GetDataModule(whichModule); techModule->IsFaction()) {
+						const DataModule::BuyMenuTheme &techBuyMenuTheme = techModule->GetFactionBuyMenuTheme();
+
+						// Match the ObjectPicker skin and background color with the BuyMenu for visual consistency.
```

Comment:
No need for this inline comment - method comments exist for a reason, use that to explain that this will match up with the buy menu for visual consistency.